src/IoC/HandlerExtensions.cs

```diff
@@ -49,7 +49,11 @@ public static IServiceCollection AddHandlers(this IServiceCollection services)
                 .AddSingleton<IHandler, JsonResultHandler<PcasStorageTypeResource>>()
                 .AddSingleton<IHandler, JsonResultHandler<IEnumerable<PcasStorageTypeResource>>>()
                 .AddSingleton<IHandler, JsonResultHandler<PcasBoardResource>>()
-                .AddSingleton<IHandler, JsonResultHandler<IEnumerable<PcasBoardResource>>>();
+                .AddSingleton<IHandler, JsonResultHandler<IEnumerable<PcasBoardResource>>>()
+                .AddSingleton<IHandler, JsonResultHandler<StoresPalletResource>>()
+                .AddSingleton<IHandler, JsonResultHandler<IEnumerable<StoresPalletResource>>>()
+                .AddSingleton<IHandler, JsonResultHandler<LocationTypeResource>>()
```

Comment:
might not need this one if you only ever run searches, i.e. request lists of LocationTypeResource's